golos.ctrl/src/golos.ctrl.cpp

```diff
@@ -55,6 +59,54 @@ void control::updateprops(properties new_props) {
     });
 }
 
+void control::transfer(account_name from, account_name to, asset quantity, string memo) {
+    if (to == _self && quantity.amount > 0) {
+        // Don't check `from` for now, just distribute to top witnesses
+
+        //Reinitialize props because called without symbol
+        _token = quantity.symbol;
+        props_tbl tbl{_self, _token};
+        auto i = tbl.begin();
+        bool exists = i != tbl.end();
```

Comment:
do we really need this variable?

Reply:
Yeah, this part can be optimized, but I just copy&pasted, because it will be removed in #283